dashboard/frontend/src/components/CreateTensorBoard.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
```

Comment:
Are you launching a separate instance of TB or is this the TB managed as part of the TfJob? If not what is the lifecycle of the TensorBoard instance?

Reply:
This are the details for the TB managed as part of the TfJob.

Reply:
Thanks.
Resolved.